src/test/integration/integrationtests.py

```diff
@@ -144,6 +147,136 @@ def test_jobs_by_osu_id(self):
         self.assertEqual(
             utils.get_person_by_osu_id(not_valid_osu_id).status_code, 404)
 
+    def test_post_job_by_osu_id(self):
```

Comment:
This method is pretty long. Could you split this up in to several different tests?